src/index.js

```diff
@@ -0,0 +1,3 @@
+import Demo from "./components/Demo";
+
+export default Demo;
```

Comment:
This demo should be in `packages/` directory

Reply:
Thanks @karimkawambwa, one little question though, I should get rid of the `src` folder, right?